publish.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+npm run build
+gem build inferno_core.gemspec
+gem push inferno_core*.gem
```

Comment:
this won't do anything wacky if old ones are left around, will it?

Reply:
It will just fail, so you need to remove the old ones.